packages/core/src/utils/validator-extras.js

```diff
@@ -87,6 +88,7 @@ validator.isNull = validator.isEmpty;
 
 // isDate removed in 7.0.0
 // https://github.com/chriso/validator.js/commit/095509fc707a4dc0e99f85131df1176ad6389fc9
+// TODO: isDate has been added back https://github.com/validatorjs/validator.js/pull/1270
```

Comment:
There is some debate on how well `isDate` functions so I wouldn't mind using a date library for this tbh

Reply:
I think I'll leave the comment for now and we'll decide what we do once we continue working on #15497